src/agent/tests/windows_service_test.cpp

```diff
@@ -0,0 +1,111 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <windows_service.cpp>
```

Comment:
Shouldn't it be the header file here that is included?

Reply:
Fixed!